starknet_py/net/account/base_account.py

```diff
@@ -51,13 +51,16 @@ async def get_nonce(self) -> int:
 
     @abstractmethod
     async def get_balance(
-        self, token_address: Optional[AddressRepresentation] = None
+        self,
+        token_address: Optional[AddressRepresentation] = None,
+        chain_id: Optional[StarknetChainId] = None,
     ) -> int:
         """
         Checks account's balance of specified token.
 
         :param token_address: Address of the ERC20 contract.
-            If not specified it will be payment token address.
+        :param chain_id: Identifier of the Starknet chain used.
+            If token_address is not specified it will be used to determine network's payment token address.
```

Comment:
it would be also irrelevant when `token_address` is provided - worth to note here